services/integration/src/models/constitution/constitution.models.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require('mongoose');
```

Comment:
You are naming this files as constitution.model`S`. there is a `S` that should not be there.